config/initializers/revision.rb

```diff
@@ -0,0 +1 @@
+Settings.revision = File.read("#{Rails.root}/REVISION").chomp if File.exists?("#{Rails.root}/REVISION")
```

Comment:
Not sure this is how others would do it, but it seems to work on stage.

https://app.honeybadger.io/projects/49092/faults/42851601#notice-context was generated from called `window.onerror()` in the browser console.

Reply:
Maybe this could just go in `config/settings.yml` so there is no confusion as to where it comes from.